style.css

```diff
@@ -0,0 +1,583 @@
+  position: relative;
+  left: -17px;
+  background-color: aliceblue;
+}
+
+.header_title h2 {
+  position: relative;
+  bottom: 55px;
+}
+
+.header_title .line {
+  position: relative;
+  bottom: 55px;
+}
+
+.recent_work_container .project_btn {
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  font-weight: bold;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+  z-index: inherit;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: repeat(6, 1fr);
+  margin-top: 20px;
+  overflow: hidden;
+  justify-items: center;
+}
+
+.box {
+  padding: 23px;
+  background-image: url('./images/placeholder.png');
+  width: 240px;
+  height: 300px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+
+  /* margin-left: 20px; */
+}
+
+.box h4 {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.box p {
+  width: 200px;
+  height: 99px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.grid button {
+  background: rgba(255, 255, 255, 0.24);
+  width: 80px;
+  color: #fff;
+  font-weight: bold;
+  position: relative;
+  top: 50px;
+  left: 1.4px;
+}
+
+.btn {
+  background-color: #ff6b00;
+  width: 300px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  text-align: center;
+  align-items: center;
+  color: #fff;
+  border: none;
+  height: 50px;
+  position: relative;
+  top: 55px;
+  right: 22px;
+}
+
+.btn:hover {
+  background-color: #1c1a19;
+  color: white;
+}
+
+.second {
+  position: relative;
+  bottom: 200px;
+}
+
+.about_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  width: 100%;
+  height: 100%;
+  background-image: url('./images/illustration-about-me-mobile.png');
+  background-repeat: no-repeat;
+  background-position: 150px 300px;
+  position: relative;
+  bottom: 270px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  align-items: flex-start;
+  width: 250px;
+  height: 100vh;
+  gap: 10px;
+}
+
+.about h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 42px;
+  color: #ff6b00;
+  margin-right: 20px;
+  position: relative;
+  left: 20px;
+}
+
+.about p {
+  width: 260px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #f4f5f7;
+  margin-right: 20px;
+  position: relative;
+  left: 20px;
```

Comment:
- [ ] Great job removing the white space to the right of your section as suggested by previous reviewers. I noticed that your About section is still pushed to the left of your page. This is because you are using a fixed width for the section and its children elements. Kindly refrain from using fixed width, use a flexible width instead in percentage`%`. Change the widths on lines 316, 333. Also, add a width of 100% to `.list` class.
### Your design
![image](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/94551484/4012a134-3ae0-4dd1-b890-19543c34fe91)
### Figma template
![image](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/94551484/8b467f4b-b470-4eca-9c92-a8fcf8a2cfde)


- [Optional] Consider moving the background image downwards so that the about me text can be visible. Also consider removing the space between the set resume button and list of technologies to match given template.